packages/evm/src/interpreter.ts

```diff
@@ -760,6 +760,18 @@ export class Interpreter {
     return baseFee
   }
 
+  /**
+   * Returns the Blob Base Fee of the block as proposed in [EIP-7516](https;//eips.etheruem.org/EIPS/eip-7516)
+   */
+  getBlobBaseFee(): bigint {
+    const blobBaseFee = this._env.block.header.getBlobGasPrice()
+    if (blobBaseFee === undefined) {
+      // Sanity check
+      throw new Error('Block has no Blob Base Fee')
+    }
+    return blobBaseFee
+  }
```

Comment:
Not so deep in the blob pricing topic: so is "blobBaseFee === blobGasPrice" here?

If so: I think we then should consider still renaming all `blobGasPrice` related naming in `Block` to `blobBaseFee`? (we can still do since all 4844 functionality is still non-final)

Reply:
Guess we might want to wait with merging here until this is settled? 🤔 Since this is the very last question on ACD channel and still un-answered?